src/schemas/ba-sak/begrunnelse/utils.ts

```diff
@@ -21,5 +22,26 @@ export const hentNasjonaltFeltRegler = (rule, feilmelding: string) =>
 export const hentNasjonalHjemmelRegler = rule =>
   hentNasjonaltFeltRegler(
     rule,
-    'En nasjonal begrunnelse-hjemel er valgt, men behandlingstema for begrunnelsen er ikke nasjonal.',
+    'En nasjonal hjemel er valgt, men behandlingstema for begrunnelsen er ikke nasjonal.',
   );
+
+export const hentEØSHjemmelRegler = rule =>
+  hentNasjonaltFeltRegler(
```

Comment:
Burde denne bytte navn? Klarer ikke helt å skjønne hvordan funksjonen hentNasjonaltFeltRegler skal funke for både EØS og nasjonal 🤔

Reply:
Oj, nei dette ble feil 🤔